open-craft/opencraft
# BB-1682: Deprecate swift features

This PR adds some deprecation warnings, comments and removes mentions about `swift` from the user interface.

**Testing instructions:**
1. Check that the tests run correctly.
2. Provision an instance with S3 storage and check that the Instance Configuration page correctly states when S3 is provisioned and deprovisioned.

**Reviewer:**
